lib/adapter/freshheads/JavascriptAdapter.ts

```diff
@@ -45,9 +44,7 @@ export default class JavascriptAdapter implements Adapter {
     ) {
         const builder = new Builder(builderConfig, webpackConfig);
 
-        builder
-            .add(new BabelLoaderAdapter(this.config.babelConfig))
-            .add(new JavascriptMinimizationAdapter());
```

Comment:
Waarom haal je die weg? Is daar nu ook een andere plugin voor?

Reply:
webpack 5 heeft zelf terser-webpack-plugin in de optimizer zitten. Vandaar dat je met '...' die default kan extenden.
Voordeel is dat we zelf nu die plugin niet meer hoeven bij te houden, dit loopt gewoon samen met de webpack 5 release.

Reply:
ok, maar je toevoeging van die `...` zit toch in de `CssMinimizationAdapter`? Lijkt me dan geen goede benaming meer..

Reply:
Denk je eerder aan een algemene MinimizationAdapter dan ? Denk inderdaad niet dat als we nog een adapter met minimization introduceren we nogmaal die `... ` kunnen gebruiken.

Reply:
prima